k8s/kubectl/admin/configmap/wmgr/etc/wmgr.cnf

```diff
@@ -0,0 +1,62 @@
+#
+# configuration file for wmgrService, production version
+#
```

Comment:
Could you also add connection parameters for the worker's XRootD/CMS service? Something like:
host = localhost
port = 1094

This would be used to manage chunk lists known to the corresponding worker.

Reply:
I think we could add this when we will support them in core/modules/wmgr/python/xrdMgr.py